set.mm

```diff
@@ -51040,64 +51108,98 @@ Contrast with domain (defined in ~ df-dm ).  For alternate definitions,
       HVDVEADRVQUBUCUDUEUKUFUGDEUTBGSDEUTBUQSUHUIUTAURULUMUNUO $.
   $}
 
-  $( Ordered pair membership in a restriction.  Exercise 13 of [TakeutiZaring]
-     p. 25.  (Contributed by NM, 14-Oct-2005.)  (Revised by BJ,
-     18-Feb-2022.) $)
-  opelresg $p |- ( B e. V -> ( <. A , B >. e. ( C |` D ) <->
-                    ( <. A , B >. e. C /\ A e. D ) ) ) $=
+  $( Ordered pair elementhood in a restriction.  Exercise 13 of [TakeutiZaring]
+     p. 25.  (Contributed by NM, 13-Nov-1995.)  (Revised by BJ, 18-Feb-2022.)
+     (Revised by Peter Mazsa, 24-Sep-2022.) $)
+  opelres $p |- ( C e. V ->
+      ( <. B , C >. e. ( R |` A ) <-> ( B e. A /\ <. B , C >. e. R ) ) ) $=
```

Comment:
What was the reason why the class variables were exchanged? Maybe I missed this part in the (long) discussion...

Reply:
@avekens : the main change is that the conjunction in the consequent is commuted. The reason is that most usages use that form, so this saves commutation theorems like ~ancom more often than it introduces them. (The change of variable names is not very important, but having `R` for what is intended to be a relation seems to be for the better.)